src/lenses/RougeScoreLens.tsx

```diff
@@ -0,0 +1,26 @@
+import { Lens } from "../types";
+import rouge from 'rouge';
+
+const calculateRougeScore=(values: unknown[])=>{
+        return rouge.n(values[0], values[1])
+    // return 0;
+}
+
+const RougeScoreLens: Lens = ({values})=>{
+    const result = calculateRougeScore(values)
+    return (<>Rouge score: {result}</>)
+}
+
+RougeScoreLens.key="RougeScoreView"
+RougeScoreLens.dataTypes=["str"]
+RougeScoreLens.defaultHeight = 48
+RougeScoreLens.minHeight=22
+RougeScoreLens.maxHeight=512
+RougeScoreLens.multi=true
+RougeScoreLens.displayName="ROUGE Score"
+RougeScoreLens.filterAllowedColumns=(allColumns, selectedColumns) => {
+    if(selectedColumns.length===2)
+        return selectedColumns
+    else return allColumns.filter(({type}) => type.kind === 'str')
+};
+export default RougeScoreLens
```

Comment:
Is it correct, that the rouge score only makes sence when at least and not more thatn two columns are selected?

You might want to look at the [AudioLense](https://github.com/Renumics/spotlight/blob/main/src/lenses/AudioLens.tsx#L56) and implement the `isSatisfied` property to check if two columns are selected.

Reply:
You are right. I've implemented isSatisfied property now.